README.md

```diff
@@ -102,8 +102,9 @@ Next let's create a `users` collection using Synth's configuration language, and
             }
         },
         "joined_on": {
-            "type": "string",
-            "date_time": {
+            "type": "object",
```

Comment:
Might make more sense to just have this as `"joined_on": { "type": "date_time", ... }` - i.e. remove the nested object.